src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Serialization/IRazorProjectInfoDeserializer.cs

```diff
@@ -8,6 +8,7 @@ namespace Microsoft.AspNetCore.Razor.LanguageServer.Serialization;
 
 internal interface IRazorProjectInfoDeserializer
 {
+    RazorProjectInfo? DeserializeFromString(string? projectInfoString);
```

Comment:
Consider naming the parameter `encodedText` or `base64String` or something like that. The notion that it's a `projectInfo` string is kind of implied by the interface.